### PR TITLE
Changed absolute links to relative links

### PR DESCRIPTION
**Pull request by bdamerac**

### CODE REVIEW

**Comment by bdamerac** [comment]
Can someone deploy it so I know what it looks like?
I still have to make some changes and I don't have pelican

**Comment by evykassirer** [comment]
sorry, was on a trip yesterday - deploying now!

File: content/S2016/S16-event-machine-learning.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ Technical Workshop Series: This event is generously sponsored by
 Technical Workshop Series. We are hosting three such workshops 
 focused on teaching technical skills in partnership with BlackBerry. 
 To learn more about our Technical Workshop Series and partnership 
-with BlackBerry see our [sponsors page](http://wics.uwaterloo.ca/sponsorship/).
+with BlackBerry see our [sponsors page]({filename}/pages/sponsorship.md).
```
Comment (evykassirer) [inline content/S2016/S16-event-machine-learning.md]: this one is broken - the path is `/sponsorship` not `/pages/sponsorship.md`

File: content/pages/sponsors.md [hunk 2]
```diff
@@ -16,5 +16,5 @@ BlackBerry has agreed to fund our three events and we are very excited to
 work with them!
 
 To view all the event posts for our technical workshops, click 
-[here](http://wics.uwaterloo.ca/tags/blackberry). In Fall 2015 we ran our 
+[here]({tag}blackberry). In Fall 2015 we ran our
```
Comment (evykassirer) [inline content/pages/sponsors.md:19]: 👍

**Comment by evykassirer** [comment]
why did you close this PR?

**Comment by bdamerac** [comment]
I messed up. Can I just make another pull request from a different branch now? (Don't really have time to learn git now and changing things is just grep and sed and effortless)

**Comment by evykassirer** [comment]
I can help you on irc if you want, but you're welcome to make a pull request form another branch

The hope is that all committee members are comfortable using git by the end of the term though, so if you can find the time to work on this with me at some point (doesn't have to be this week) that's be preferable

**Comment by bdamerac** [comment]
Thanks!

**Comment by evykassirer** [comment]
I think you've fixed it now :) just reopen the pull request and we can see how it's looking

**Pull request closed without merging**